entities/src/main/java/com/flipkart/varadhi/entities/VaradhiEntity.java

```diff
@@ -0,0 +1,20 @@
+package com.flipkart.varadhi.entities;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.FieldDefaults;
+
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@Getter
+public abstract class VaradhiEntity {
```

Comment:
There is already class VaradhiResource with similar purpose. Does this have different purpose ?